django_reactive/templates/django_reactive.html

```diff
@@ -1,3 +1,9 @@
+<pre>
+{% if errors %}
```

Comment:
Not very familiar with the React form generation lib so I just put the errors here, but maybe we can pass them and have them rendered against the individual fields somehow.